test/tools/ossfuzz/abiV2Proto.proto

```diff
@@ -78,6 +54,16 @@ message DynamicByteArrayType {
     required DType type = 1;
 }
 
+message ArrayType {
+    required Type t = 1;
+    required uint32 length = 2;
+    required bool is_static = 3;
```

Comment:
I wonder it's nicer to replace the two required ``length`` and ``is_static`` by an optional ``length`` only (it being present meaning static, it not being present meaning dynamic). But it's fine as it is a well.

Reply:
Ah right! It's a good suggestion but might warrant another PR though

Reply:
Yes, no need for this PR.